docs/organizations/using-a-coding-standard.md

```diff
@@ -86,6 +86,8 @@ To edit an existing coding standard or change the repositories that follow that
 
     Codacy will start using the updated coding standard on the next analysis of each selected repository.
 
+If you stopped applying the coding standard to any repository, Codacy will restore the previous code pattern configurations for that repository.
```

Comment:
I wonder if we should give more visibility to this part using an `important` admonition (like we had for the previous warning that we're now removing).

Also, consider (with the team) if we should make it more explicit that Codacy restores the code pattern configurations **only** but not the tool status. :thinking: